apps/mdn/mdn-aws/k8s/regions/oregon/stage.mm.sh

```diff
@@ -53,7 +53,8 @@ export WEB_NAME=web
 export WEB_REPLICAS=1
```

Comment:
We're missing:
```sh
 export ELB_CONNECTION_DRAINING_ENABLED=True
```
in this file.

Reply:
Good catch, fixed in ad410a1b368806701ade90f364d2c4fe859e2cd4